src/interface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+interface IShapeDrawRectangle
```

Comment:
I think IShapeDrawRectangle and IShapeDrawSquare could be just one interface IDrawable with function draw();

Reply:
I added IDrawable for draw()